Add compatibility with hierarchicalMenu/breadcrumb/RatingsMenu in readme (#682)
meilisearch/instant-meilisearch: README.md

})
```

### ✅ HierarchicalMenu

[Hierarchical menu references](https://www.algolia.com/doc/api-reference/widgets/hierarchical-menu/js/)

> The `hierarchicalMenu` widget is used to create a navigation based on a hierarchy of facet attributes. It is commonly used for categories with subcategories. See [usage](#hierarchical-menu-usage) bellow.

Reviewer: ```suggestion
The `hierarchicalMenu` widget is used to create navigation based on a hierarchy of facet attributes. It is commonly used for categories with subcategories. See [usage](#hierarchical-menu-usage) below.
```

- ✅ container: The CSS Selector or HTMLElement to insert the refinements. _required_
- ✅ attribute: The name of the attributes to generate the menu with. _required_.

> - ✅ limit: How many facet values to retrieves.

Reviewer: ```suggestion
- ✅ limit: How many facet values to retrieve.
```

- ✅ showMore: Whether to display a button that expands the number of items.
- ✅ showMoreLimit: The maximum number of displayed items (min `2`).
- ❌ separator: The level separator used in the records. (default `>`).
- 🤷‍♀️ rootPath: The prefix path to use if the first level is not the root level.
- ❌ showParentLevel: Whether to show the siblings of the selected parent level of the current refined value.
- ✅ sortBy: How to sort refinements. [See guide](https://www.algolia.com/doc/api-reference/widgets/hierarchical-menu/js/#widget-param-sortby)
- ✅ templates: The templates to use for the widget.- ✅ templates: The templates to use for the widget.
- ✅ cssClasses: The CSS classes to override.

#### Hierarchical Menu Usage

> To make it work with Meilisearch your documents must have a specific structure, explaination on the structure can [be found here](https://www.algolia.com/doc/api-reference/widgets/hierarchical-menu/js/#requirements).

Reviewer: ```suggestion
To make it work with Meilisearch your documents must have a specific structure, an explanation of the structure can [be found here](https://www.algolia.com/doc/api-reference/widgets/hierarchical-menu/js/#requirements).
```

> Contrary to `instantsearch.js`, the hierchical fields are added in [`filterableAttributes`](https://docs.meilisearch.com/reference/api/filterable_attributes.html#update-filterable-attributes).

Reviewer: ```suggestion
Contrary to `instantsearch.js`, the hierarchical fields are added in [`filterableAttributes`](https://docs.meilisearch.com/reference/api/filterable_attributes.html#update-filterable-attributes).
```
